runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -611,6 +611,35 @@ public void onCommandExecuted(CommandExecuted commandExecuted)
 				updateClue(clueScroll);
 			}
 		}
+		else if (developerMode && commandExecuted.getCommand().equals("hotcold"))
```

Comment:
The `developerMode` check should be a guard clause, and extracted from each of these `if` conditions.